batched/dense/unit_test/Test_Batched_SerialTbsv.hpp

```diff
@@ -0,0 +1,342 @@
+//@HEADER
+// ************************************************************************
+//
+//                        Kokkos v. 4.0
+//       Copyright (2022) National Technology & Engineering
+//               Solutions of Sandia, LLC (NTESS).
+//
+// Under the terms of Contract DE-NA0003525 with NTESS,
+// the U.S. Government retains certain rights in this software.
+//
+// Part of Kokkos, under the Apache License v2.0 with LLVM Exceptions.
+// See https://kokkos.org/LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//@HEADER
+/// \author Yuuichi Asahi (yuuichi.asahi@cea.fr)
+#include <gtest/gtest.h>
+#include <Kokkos_Core.hpp>
+#include <Kokkos_Random.hpp>
+
+#include "KokkosBatched_Util.hpp"
+#include "KokkosBatched_Tbsv.hpp"
+#include "Test_Utils.hpp"
+
+using namespace KokkosBatched;
+
+namespace Test {
+namespace Tbsv {
+
+template <typename U, typename T, typename D>
+struct ParamTag {
+  using uplo  = U;
+  using trans = T;
+  using diag  = D;
+};
+
+template <typename DeviceType, typename AViewType, typename BViewType,
+          typename ScalarType, typename ParamTagType, typename AlgoTagType>
+struct Functor_BatchedSerialTrsv {
+  using execution_space = typename DeviceType::execution_space;
+  AViewType _a;
+  BViewType _b;
+
+  ScalarType _alpha;
+
+  KOKKOS_INLINE_FUNCTION
+  Functor_BatchedSerialTrsv(const ScalarType alpha, const AViewType &a,
+                            const BViewType &b)
+      : _a(a), _b(b), _alpha(alpha) {}
+
+  KOKKOS_INLINE_FUNCTION
+  void operator()(const ParamTagType &, const int k) const {
+    auto aa = Kokkos::subview(_a, k, Kokkos::ALL(), Kokkos::ALL());
+    auto bb = Kokkos::subview(_b, k, Kokkos::ALL());
+
+    KokkosBatched::SerialTrsv<
+        typename ParamTagType::uplo, typename ParamTagType::trans,
+        typename ParamTagType::diag, AlgoTagType>::invoke(_alpha, aa, bb);
+  }
+
+  inline void run() {
+    using value_type = typename AViewType::non_const_value_type;
+    std::string name_region("KokkosBatched::Test::SerialTrsv");
```

Comment:
Maybe rename this to avoid confusion
```suggestion
    std::string name_region("KokkosBatched::Test::SerialTbsv");
```